### ISSUE TITLE
Folder Upload

### ISSUE THREAD

**Issue by theredkoala21** [Ylianst/MeshCentral#396]
Just tried to upload a folder via the Files section of mesh. 
Via the upload button, it does not support the selection of folders, only files.

But you can drag and drop folders from your desktop into the remote Agent files section. This pops up with the prompt like it is going to upload the folder, but sits there forever.

Can you add a function for uploading folders + contents of folders.
I can get around it by zipping, uploading and unzipping. 
But it would be really handy.

Thanks

**Comment by Ylianst** [comment]
Agreed. Folder upload is currently not supported and not handled well, however it's certainly something that can be added.

**Comment by Ylianst** [comment]
I looked into this and when I drag & drop a folder into the browser, I don't get access to all of the files in that folder. So I can't update the folder content. I published MeshCentral v0.3.9-a to fix problems associated with dropping a folder in the upload windows, but it will just ignore these entries. It seems like the best I can do unless we find other tricks.

Bryan at some point may add support for folder download, but we would "tar" the files and send them down as a single big file.
